json.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+module.exports = {
+  extends: ['plugin:jsonc/base'],
+  overrides: [
+    {
+      files: ['package.json', 'package-lock.json'],
```

Comment:
Wouldn't this lint `package-lock.json` file? It is an auto-generated file. Do you see a scenario where this needs to be linted?

Reply:
The `overrides.files` here is to prevent `jsonc/sort-keys` to be applied to `package.json` and `package-lock.json`.

Reply:
right, but my question is: Should we lint `package-lock.json` _at all_ ? As it is now, if `plugin:jsonc/base` includes other rules, they will be checked against `package-lock.json`.

Perhaps it should go into `excludedFiles`. Thoughts?